api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3693,6 +3695,81 @@ message ElasticsearchRequest {
   string Query = 12 [(gogoproto.jsontag) = "query"];
 }
 
+// DynamoDBRequest is emitted when a user executes a DynamoDB request.
+// We support DynamoDB access via app access and database access,
+// therefore this event includes either DatabaseMetadata or AppMetadata.
+message DynamoDBRequest {
```

Comment:
We need to introduce event name and event code for `DynamoDBRequest` audit event.

Reply:
Yep, will be added in a later PR, this one is just to get the protos definition nailed down